www/js/controllers.js

```diff
@@ -8,18 +8,20 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.survey.launch',
                                         'emission.stats.clientstats',
                                         'emission.incident.posttrip.prompt',
-                                        'customURLScheme'])
+                                        'customURLScheme',
+                                        'emission.splash.ionicNotification'])
 
 .controller('RootCtrl', function($scope) {})
 
 .controller('DashCtrl', function($scope) {})
 
 .controller('SplashCtrl', function($scope, $state, $interval, $rootScope, 
     CustomURLScheme, UpdateCheck, StartPrefs, ReferralHandler, PushNotify,
-    LocalNotify, ClientStats, PostTripAutoPrompt, SurveyLaunch)  {
+    LocalNotify, ClientStats, PostTripAutoPrompt, SurveyLaunch, ionicNotification)  {
   console.log('SplashCtrl invoked');
   // alert("attach debugger!");
   // PushNotify.startupInit();
+  ionicNotification.redirectNotification();
```

Comment:
the name sounds like you are generating a redirect notification instead of listening to it.
better to initialize within the file by listening to the ready notification (see the local notification example)